pyrsa/rdm/__init__.py

```diff
@@ -0,0 +1 @@
+from pyrsa.data.rdm import RDMs
```

Comment:
I think this should be `from pyrsa.rdm.rdms import RDMs`

Reply:
fixed and pushed.